go.mod

```diff
@@ -4,9 +4,9 @@ go 1.23.9
 
 require (
 	github.com/VictoriaMetrics/fastcache v1.12.1
-	github.com/ava-labs/avalanchego v1.13.3-rc.1
+	github.com/ava-labs/avalanchego v1.13.2-rc.0
```

Comment:
I don't know how to get the libevm version updated without downgrading avalanchego...